src/main/java/Domus/Main.java

```diff
@@ -114,41 +112,20 @@ public static void main(String[] args) throws IOException {
 
         // check accuracy, precision and recall
         // for now it checks all 5 days of the missing users, parameters to tune...
-        int samples = (6 - nUsers) * 5 * 3;
-        int truePos = 0;
-        int trueNeg = 0;
-        int falsePos = 0;
-        int falseNeg = 0;
+        PerformanceEvaluator performanceEvaluator = new PerformanceEvaluator(result);
```

Comment:
Tutta questa parte da 115 a 128 le metterei tra gli ExperimentUtils nella funzione di log, o in una nuova, almeno rimane più pulito

Reply:
Ci sta

Reply:
Forse devo ripensare a come gestire i vari experiment, che ogni volta bisogna modificare ogni file se si aggiunge qualcosa

Reply:
Faccio quella modifica poi uniamo i branch così possiamo testare bene tutto, ok?

Reply:
Va bene, allora aspetto a spostare quella parte del main nelle utils